premailer/__main__.py

```diff
@@ -140,7 +145,7 @@ def main(args):
         disable_basic_attributes=options.disable_basic_attributes,
         disable_validation=options.disable_validation
     )
-    options.outfile.write(p.transform(pretty_print=options.pretty))
+    options.outfile.write(p.transform(encoding=options.encoding, pretty_print=options.pretty))
```

Comment:
CI fails because this line is not PEP8 compliant. You need to re-write it so it's not longer than 80 characters. 
Change it to be indented like this:
```python
options.outfile.write(p.transform(
    encoding=options.encoding, 
    pretty_print=options.pretty
))
```